src/layout/Grid.js

```diff
@@ -87,7 +87,9 @@
         return this;
     };
 
-    Grid.prototype.getCell = function (row, col) {
+    Grid.prototype.getCell = function (_row, _col) {
+        var row = _row * this.cellDensity();
+        var col = _col * this.cellDensity();
```

Comment:
@GordonSmith This might cause undesired effects... this assumes that _row and _col aren't already adjusted for cellDensity.